actors.md

```diff
@@ -4,12 +4,32 @@ Any implementations of the Filecoin actors must be exactly byte for byte compati
 
 This spec describes a set of actors that operate within the [Filecoin State Machine](state-machine.md). All types are defined in [the basic type encoding spec](data-structures.md#basic-type-encodings).
 
-- [Init Actor](#init-actor)
-- [Storage Market Actor](#storage-market-actor)
-- [Storage Miner Actor](#storage-miner-actor)
-- [Payment Channel Broker Actor](#payment-channel-broker-actor)
+## Actor State
+
+These below used `kindeded` representation, as the type can be inferred from the context, in which
+they are used (`Actor` or `UnsignedMessage`).
+
+```sh
+type ActorState union {
+    | InitActorState
+    | AccountActorState
+    | StorageMarketActorState
+    | StorageMinerActorState
+    | PaymentChannelBrokerActorState
+    | MultisigActorState
+} representation kinded
```

Comment:
I don't think these can be kinded; and this is incomplete either way.

To use kinded unions, each member must have a distinct representation kind -- so for example, you can't use two different types which are both represented as map, nor two different types represented as lists, and so on.

If you have two structs, it's still possible to have them be discriminated in kinded union if one has `representation map` (reprkind==map) and the other has `representation tuple` (reprkind==list).  But those are just details to the above rule.

All members in a kinded union have to have a kind keyword after them, in order to make all this locally lexically clear.  More than one of these types would have the kind "map", and so something's fishy here.

(Long story short, you probably just mean to use `representation keyed` here, or something.)